server/routes/main/index.js

```diff
@@ -18,6 +18,12 @@ module.exports = {
       middleware.setUserIfTokenExists,
       require("./get-involved").bind(app, config));
 
+    // Refresh Editor Page
+    app.get("/refresh",
+      middleware.clearRedirects,
+      middleware.setUserIfTokenExists,
```

Comment:
Do we need this (i.e., any user token stuff)?  Since this lives in the Brackets iframe, all user account stuff should *not* be part of this for security reasons.